attest/verifier/config/src/lib.rs

```diff
@@ -332,61 +350,50 @@ mod tests {
-            product_id: 3,
-            minimum_svn: 4,
-            mitigated_config_advisories: vec!["FOO".to_string()],
-            mitigated_hardening_advisories: vec![
-                "INTEL-SA-00334".to_string(),
-                "INTEL-SA-00615".to_string(),
-            ],
-        };
+        let expected_fog_view = TrustedMeasurement::from(TrustedMrSignerMeasurement::new(
+            &hex!("2c1a561c4ab64cbc04bfa445cdf7bed9b2ad6f6b04d38d3137f3622b29fdb30e"),
+            3,
+            4,
+            ["INTEL-SA-00334", "INTEL-SA-00615"],
+        ));
         assert_eq!(v3_fog_view, &expected_fog_view);
 
         let v4 = tms.table.get("v4").unwrap();
         assert_eq!(v4.len(), 4);
         let v4_consensus = v4.get("consensus").unwrap();
-        let expected_consensus = StatusVerifierConfig::Mrenclave {
-            mr_enclave: hex!("e35bc15ee92775029a60a715dca05d310ad40993f56ad43bca7e649ccc9021b5"),
-            mitigated_config_advisories: vec!["FOO".to_string()],
-            mitigated_hardening_advisories: vec![
-                "INTEL-SA-00334".to_string(),
-                "INTEL-SA-00615".to_string(),
-                "INTEL-SA-00657".to_string(),
-            ],
-        };
+        let expected_consensus = TrustedMeasurement::from(TrustedMrEnclaveMeasurement::new(
+            &hex!("e35bc15ee92775029a60a715dca05d310ad40993f56ad43bca7e649ccc9021b5"),
+            ["INTEL-SA-00334", "INTEL-SA-00615", "INTEL-SA-00657"],
+        ));
         assert_eq!(v4_consensus, &expected_consensus);
 
         let v4_fog_view = v4.get("fog-view").unwrap();
-        let expected_fog_view = StatusVerifierConfig::Mrenclave {
-            mr_enclave: hex!("8c80a2b95a549fa8d928dd0f0771be4f3d774408c0f98bf670b1a2c390706bf3"),
-            mitigated_config_advisories: vec!["FOO".to_string()],
-            mitigated_hardening_advisories: vec![
-                "INTEL-SA-00334".to_string(),
-                "INTEL-SA-00615".to_string(),
-                "INTEL-SA-00657".to_string(),
-            ],
-        };
+        let expected_fog_view = TrustedMeasurement::from(TrustedMrEnclaveMeasurement::new(
+            &hex!("8c80a2b95a549fa8d928dd0f0771be4f3d774408c0f98bf670b1a2c390706bf3"),
+            ["INTEL-SA-00334", "INTEL-SA-00615", "INTEL-SA-00657"],
+        ));
         assert_eq!(v4_fog_view, &expected_fog_view);
 
-        let _ = tms.create_verifier("consensus").unwrap();
-        let _ = tms.create_verifier("fog-ingest").unwrap();
-        let _ = tms.create_verifier("fog-ledger").unwrap();
-        let _ = tms.create_verifier("fog-view").unwrap();
+        assert_eq!(
+            tms.measurements("consensus").unwrap(),
+            vec![v3_consensus.clone(), v4_consensus.clone()]
+        );
+        assert_eq!(
+            tms.measurements("fog-view").unwrap(),
+            vec![v3_fog_view.clone(), v4_fog_view.clone()]
+        );
 
-        assert!(tms.create_verifier("impostor").is_err());
+        assert!(matches!(
```

Comment:
there is an `assert_matches!` macro i think that gives slightly better output, but it requires pulling in another crate

:shrug:

Reply:
updated to `assert_matches!`